pipeline/rating/tasks/task-helm-lint.yaml

```diff
@@ -0,0 +1,26 @@
+apiVersion: tekton.dev/v1beta1
+kind: Task
+metadata:
+  name: helm-lint
+spec:
+  params:
+  - name: url
+    type: string
+  - name: component
+    type: string
+  - name: version
+    type: string
+  results:
+  - name: LINT
+    type: string
+  steps:
+    - name: helm-lint
+      image: kubebb/component-rbac-lint:v0.0.1
+      script: |
+        #!/usr/bin/env bash
+        # Download and untar chart package
+        helm pull $(params.url)
+        tar -zxvf $(params.component)-$(params.version).tgz
```

Comment:
use `--untar`?
`--untar                      if set to true, will untar the chart after downloading it`